services/src/messaging/src/test/java/org/openkilda/messaging/model/LinkPropsMaskTest.java

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.model;
+
+import org.openkilda.messaging.StringSerializer;
+import org.openkilda.messaging.Utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+
+public class LinkPropsMaskTest implements StringSerializer {
```

Comment:
StringSerializer is used here only for keeping serialized value. I believe local var is easier and fits the purpose better.